src/common/snippets/src/pass/collapse_subgraph.cpp

```diff
@@ -114,14 +117,19 @@ auto is_supported_op(const std::shared_ptr<const Node> &n) -> bool {
             || ov::is_type<ngraph::op::v4::Swish>(n)
             || ov::is_type<ngraph::op::v4::HSwish>(n);
     };
-    return is_supported_fq_op(n) || is_supported_unary_eltwise_op(n) || is_supported_binary_eltwise_op(n);
+    return is_supported_fq_op(n) || is_supported_unary_eltwise_op(n) || is_supported_binary_eltwise_op(n) ||
```

Comment:
Oh what's about to check for `is_supported_binary_eltwise_op` or `is_supported_unary_eltwise_op` firstly? It's a common and frequent case in Snippets. I hope it will help to reduce load time. What do you think?